editor/editor_node.h

```diff
@@ -153,6 +153,7 @@ class EditorNode : public Node {
 		FILE_CLOSE_ALL,
 		FILE_CLOSE_ALL_AND_QUIT,
 		FILE_CLOSE_ALL_AND_RUN_PROJECT_MANAGER,
+		FILE_CLOSE_ALL_AND_RUN_RELOAD_CURRENT_PROJECT,
```

Comment:
This should likely be:
```suggestion
		FILE_CLOSE_ALL_AND_RELOAD_CURRENT_PROJECT,
```

The actions are "quit", "run project manager", and "reload current project".

Reply:
It's was RUN_RELOAD_CURRENT_PROJECT, anyway I will change it.

Reply:
What I meant is that "run reload" is two verbs and sounds weird. If the action is that the edit is quit and the current project is reloaded, the "run" verb doesn't need to be included.

Reply:
Got it, does everything seem fine now or do I need to make any changes?